### ISSUE TITLE
List on probot apps page

### ISSUE THREAD

**Issue by philip-gai** [philip-gai/announcement-drafter#50]
https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md#adding-your-app

**Comment by philip-gai** [comment]
Listing: https://github.com/philip-gai/probot.github.io/blob/1c4a84054168d88c40ee9de869646acf9d699700/_apps/repost.md

**Comment by philip-gai** [comment]
Requires tests, therefore this depends on #18

**Comment by philip-gai** [comment]
PR: https://github.com/probot/probot.github.io/pull/382

**Comment by philip-gai** [comment]
New PR with new name: https://github.com/probot/probot.github.io/pull/385
